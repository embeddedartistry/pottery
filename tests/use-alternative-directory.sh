pottery init alternative-dir
pottery note "First note!"

ls .
cat alternative-dir/1992/1992-01/1992-01-12T16:34:23Z_note_FAKEUNIQUEID.md
