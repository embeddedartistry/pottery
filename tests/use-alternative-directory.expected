pottery init alternative-dir
pottery post "First post!"

ls .
alternative-dir
ls -R alternative-dir
1992

alternative-dir/1992:
1992-01

alternative-dir/1992/1992-01:
1992-01-12T16:34:23Z.md
cat alternative-dir/1992/1992-01/1992-01-12T16:34:23Z.md
Time: 1992-01-12 16:34

First post!
